client/reader/following/use-subscriptions.ts

```diff
@@ -0,0 +1,42 @@
+import { SubscriptionManager } from '@automattic/data-stores';
+import { useMemo, useEffect } from 'react';
+
+export function useSubscriptions() {
```

Comment:
It will be useful if we add a JSDoc comment for this function.

Reply:
Thanks for that feedback, added in https://github.com/Automattic/wp-calypso/pull/97030/commits/a9a533ca86b75b45601f7c12b83be29ce170e6e4